.travis.yml

```diff
@@ -3,12 +3,12 @@ language: ruby
 jobs:
   include:
     - stage: test-2.4
-      rvm: 2.4.3
+      rvm: 2.4.4
       script:
         - bundle exec rake spec
 
     - stage: test-2.5
-      rvm: 2.5.0
+      rvm: 2.5.1
```

Comment:
alternately, we could leave the patch version and always get the latest.

Reply:
seems reasonable to me (to always test patch versions)